pom.xml

```diff
@@ -67,7 +67,7 @@
         <google.re2j>1.7</google.re2j>
 
         <!-- temporary version waiting for real automatic release in ecocode repository -->
-        <ecocode-rules-specifications.version>1.5.0</ecocode-rules-specifications.version>
+        <ecocode-rules-specifications.version>1.5.1-SNAPSHOT</ecocode-rules-specifications.version>
```

Comment:
Maybe we can wait for the next version of the artifact instead of using a snapshot here?

Reply:
waiting for approvement of this implementation before releasing ecocode-rules-specifications